xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -1391,8 +1391,25 @@ bool CDVDVideoCodecAndroidMediaCodec::ConfigureMediaCodec(void)
 
   if (CJNIBase::GetSDKVersion() >= 24)
   {
-    if (m_hints.colorRange != AVCOL_RANGE_UNSPECIFIED)
-      mediaformat.setInteger(CJNIMediaFormat::KEY_COLOR_RANGE, m_hints.colorRange);
+    switch (m_hints.colorRange)
+    {
+      case AVCOL_RANGE_UNSPECIFIED:
+        CLog::Log(LOGWARNING, "CDVDVideoCodecAndroidMediaCodec::ConfigureMediaCodec Color range: "
```

Comment:
This is probably the most common for 90% of media files. For this reason I think it makes little sense to issue a WARNING in log  (unspecified --> Do nothing , all is good)

Reply:
In the tests I did of starting about 30 videos (mostly HEVC, some AVC and one VP9) only one had the unspecified value.

Although I do not see any problem in changing the log level to DEBUG in this case.

Reply:
This is much more frequent in older videos: H.264, VC1, MPEG2...